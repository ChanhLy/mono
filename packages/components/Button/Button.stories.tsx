import { Meta } from '@storybook/react';
import { Button } from './Button';

const meta: Meta = {
  title: 'Button',
  component: Button,
};
export default meta;

export function Default() {
  return <Button>Boop</Button>;
}
