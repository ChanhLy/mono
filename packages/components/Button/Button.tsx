export interface ButtonProps{
}

export function Button(props: React.PropsWithChildren<ButtonProps>) {
  const { children } = props;

  return <button type="button">{children}</button>;
}
