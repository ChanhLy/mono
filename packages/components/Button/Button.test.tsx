import renderer from 'react-test-renderer';
import { Button } from './Button';

describe('Button Component', () => {
  it('should render', () => {
    const component = renderer.create(<Button>Click me</Button>);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
