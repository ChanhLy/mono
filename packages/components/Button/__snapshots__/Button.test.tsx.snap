// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component should render 1`] = `
<button
  type="button"
>
  Click me
</button>
`;
